tests/lib/rules/forward-ref-uses-ref.js

```diff
@@ -0,0 +1,232 @@
+/**
+ * @fileoverview Ensure symmetric naming of setState hook value and setter variables
+ * @author Duncan Beevers
+ */
+
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../../../lib/rules/forward-ref-uses-ref');
+const parsers = require('../../helpers/parsers');
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+});
+
+const tests = {
+  valid: parsers.all([
+    {
+      code: `
+        import { forwardRef } from 'react'
+        forwardRef((props, ref) => {
+          return null
```

Comment:
```suggestion
          return null;
```
let's consistently use semis in all these test cases